source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -148,6 +147,9 @@ void Filter::closeStream() {
     if (grpc_service_.has_envoy_grpc()) {
       logging_info_->setBytesSent(stream_->streamInfo().bytesSent());
       logging_info_->setBytesReceived(stream_->streamInfo().bytesReceived());
+      if (!logging_info_->upstreamHost()) {
```

Comment:
nit: as you move it to closeStream(), you don't need this check anymore since this will only be called once.